lua/starfall/libs_sh/players.lua

```diff
@@ -85,16 +85,6 @@ instance:AddHook("initialize", function()
 	getent = instance.Types.Entity.GetEntity
 end)
 
-if SERVER then
-	instance:AddHook("deinitialize", function()
-		for k, ply in pairs(player.GetAll()) do
-			if SF.IsHUDActive(instance.entity, ply) then
-				ply:SetViewEntity()
-			end
-		end
-	end)
-end
```

Comment:
This was actually needed, because fun fact, Starfall doesn't actually disconnect you from a HUD when the chip is removed lol.
Maybe do this instead:  
`lua/starfall/libs_sh/builtins.lua:47`
```lua
if SERVER then
	instance:AddHook("deinitialize", function()
		for _, ply in pairs(player.GetAll()) do
			if SF.IsHUDActive(instance.entity, ply) then
				SF.EnableHud(ply, instance.entity, nil, false, true)
			end
		end
	end)
end
```

Reply:
Could also send a signal that doesn't rely on chip being valid, solely to inform players that HUD was disconnected, since right now they won't see the `Starfall HUD disconnected by Name.` chat message.